### DISCUSSION TITLE
Duplicate Feature

### DISCUSSION THREAD

**Discussion by mirrorclone** [m4mallu/clonebot#4] [General]
-feature I want
-Eaxample A channel to B channel Copy
-When copying from channel A to channel B, first write the document of the files in channel B.
-After checking the files on channel b, delete the duplicate so that it does not send the same files from channel a to channel b

![Adsız](https://user-images.githubusercontent.com/86728257/123976199-a3376280-d9c6-11eb-9de1-f3e2f4838c0c.png)

**Comment by m4mallu** [comment]
The pictorial representation is terrible! 
Although the concept is logical, it needs some serious works to do to achieve the goal.
I Hope, someone will logically push the same...
